R/display.R

```diff
@@ -72,3 +72,21 @@ display_svg = function(data = NULL, file = NULL, width = NULL, height = NULL) {
     if (!is.null(height)) metadata$height <- height
     display1('image/svg+xml', data, metadata)
 }
+
+#'Display an object using any available reprs
+#'
+#' @param obj The object to be displayed
+#' @importFrom repr mime2repr repr_text
```

Comment:
@flying-sheep you mean like this? Does it strictly need to be on two lines? I think I've seen it like this elsewhere, but maybe not. I did use roxygen.